pkg/cmd/server/api/types.go

```diff
@@ -83,6 +95,8 @@ const (
 	ControllersAll = "*"
 )
 
+type FeatureSet map[OpenShiftFeature]util.Empty
```

Comment:
I've removed AE abbreviations. I'm working on some logic for following symlinks and checking real binary name. I can add it to this PR later. Or submit it in a new one because it's not required to disable web console.